backend/app/main/views.py

```diff
@@ -149,3 +156,17 @@ def get_client_notes(id):
 
     # Failed to read json
     return "Failed to get client's notes", 400
+
+
```

Comment:
Add comment to explain what this function does + which parameters it requires in the POST request body